### DISCUSSION TITLE
Multiple API end points for single service

### DISCUSSION THREAD

**Discussion by eramudeep** [feathersjs/feathers#3368] [Q&A]
I am new to `feathersjs` i am very much confused about the endpoints. 
Consider I created a Service Named `game` now I have one endpoint which is `game` which will support `'find', 'get', 'create', 'patch', 'remove'`
But i wanted to create an endpoint like `game/active-game` and fetch only one record which has `status:'active'`
i am very much confused :(

**Comment by Complexlity** [comment]
Hello @eramudeep , you can get `active-game` via the `get` method. After getting it then you call fetch the record using the id and throw some kind of error if the id passed is not `active-game`

![image](https://github.com/feathersjs/feathers/assets/105590967/ec8b609b-374a-4528-9d6a-acd96bc64c6b)
Image from [https://feathersjs.com/guides/basics/services.html](https://feathersjs.com/guides/basics/services.html)

The get method contains an `Id` argument which represents any more query params after the service name
![image](https://github.com/feathersjs/feathers/assets/105590967/3180ce24-8488-4dc0-add2-c905e9b9d754)
Image from [https://feathersjs.com/api/services.html](https://feathersjs.com/api/services.html)

**Comment by noor-tg** [comment]
Simply use find method.  With query parameter status: 'active'
For example
```ts
users.find({
    query: {
        status: 'active'
    }
})
```

**Comment by fxpoet** [comment]
Since there is only one get method, I think it is sometimes necessary to maintain a simple structure.

I am using it this way.

`export const boardsConfigure = (app: Application) => {

    const svc = new BoardsService(app, { 
        collection: 'boards',
        operators: ['$regex', '$options']
    })

    app.use('boards', svc)
    app.service('boards').hooks(boardHooks)

    app.use('boards/like', { create: svc.like.bind(svc) })
    app.service('boards/like').hooks(boardLikeHooks)
    
    app.use('boards/comment', { create: svc.comment.bind(svc) })
    app.service('boards/comment').hooks(boardHooks) // Reuse existing hooks.
}
`
